account_due_list/views/payment_view.xml

```diff
@@ -53,7 +52,6 @@
                 <group expand="0" string="Group By...">
                     <filter string="Partner" icon="terp-partner" domain="[]" context="{'group_by':'partner_id'}"/>
                     <filter string="Invoice" icon="terp-folder-orange" domain="[]" context="{'group_by':'stored_invoice_id'}"/>
-                    <filter string="Due date" icon="terp-go-today" domain="[]" context="{'group_by':'day'}"/>
```

Comment:
This filter is userfull, you can replace for this:

 <filter string="Due date" icon="terp-go-day" domain="[]" context="{'group_by':'date_maturity:day'}"/>